cmd/ddev/cmd/config.go

```diff
@@ -268,7 +268,7 @@ func init() {
 	ConfigCommand.Flags().BoolVar(&dbWorkingDirDefaultArg, "db-working-dir-default", false, "Unsets a db service working directory override")
 	ConfigCommand.Flags().BoolVar(&dbaWorkingDirDefaultArg, "dba-working-dir-default", false, "Unsets a dba service working directory override")
 	ConfigCommand.Flags().BoolVar(&workingDirDefaultsArg, "working-dir-defaults", false, "Unsets all service working directory overrides")
-	ConfigCommand.Flags().BoolVar(&mutagenEnabled, "mutagen-enabled", false, "enable mutagen asynchronous update of project in web container")
+	ConfigCommand.Flags().StringVar(&mutagen, "mutagen", "", "\"on\" enables or \"off\" disables Mutagen asynchronous update of project in web container")
```

Comment:
I like this... but I do very much hate to change commands, even on major versions.

Reply:
Reverted